modules/base/include/openma/base/node.h

```diff
@@ -42,6 +42,7 @@
 #include "openma/base/macros.h" // _OPENMA_NOEXCEPT
 #include "openma/base/nodeid.h" // Macro OPENMA_DECLARE_NODEID used by inheriting classes.
 
+#include <cmath>
```

Comment:
The header files composing the public API should only include the minimum set of files to reduce their dependency and ensure ABI compatibility between different versions of the library. Thus, the header `cmath` should be directly included in the source files (like in `_body.cpp`, `dempstertable.cpp`, and certainly `timesequence.cpp`). Can you adapt your PR in consequence?